packages/build/package.json

```diff
@@ -23,7 +23,12 @@
   "scripts": {
     "prepublishOnly": "cd ../../ && npm run prepublishOnly",
     "prebuild": "rm -rf lib",
-    "build": "cp -a src lib/"
+    "build": "cp -a src lib/",
+    "test:dev": "run-s test:dev:*",
```

Comment:
this is super redundant for me why do we need the `run-s` here?

I think it should be like that:

```json
{
  "test:dev": "ava",
  "test:measure": "node tools/tests_duration.mjs",
  "test:ci": "c8 -r lcovonly -r text -r json ava"
}
```

Reply:
> this is super redundant for me why do we need the `run-s` here?
> 
> I think it should be like that:
> 
> ```json
> {
>   "test:dev": "ava",
>   "test:measure": "node tools/tests_duration.mjs",
>   "test:ci": "c8 -r lcovonly -r text -r json ava"
> }
> ```

@jobala the `test:ci` is not implemented like that still on the root?